azure/src/test/java/com/microsoft/azure/AzureTests.java

```diff
@@ -76,37 +72,6 @@ public void setup() throws Exception {
     }
 
     /**
-     * Tests the public IP address implementation from an individual service client
-     * @throws Exception 
-     */
-    @Test
-    public void testPublicIpAddressesInGroup() throws Exception {
```

Comment:
Note that we do have a proper, full blown e2e test for PIPs - see TestPublicIpAddress -- like the others I've written.
This was just an early test for the InGroup specifically.